src/functionalTest/resources/scenarios/RIA-1432-ftpa-applicant-appeal-decision-granted.json

```diff
@@ -0,0 +1,83 @@
+{
+  "description": "RIA-1432 FTPA Applicant Application Decision Granted",
+  "request": {
+    "uri": "/asylum/ccdAboutToSubmit",
+    "credentials": "Judge",
+    "input": {
+      "id": 1001,
+      "eventId": "ftpaDecisionLeadershipJudge",
+      "state": "ftpaSubmitted",
+      "caseData": {
+        "template": "minimal-appeal-submitted.json",
+        "replacements": {
+          "ftpaAppellantDecisionOutcomeType": "granted",
+          "ftpaAppellantSubmitted": "Yes",
+          "ftpaRespondentSubmitted": "No",
+          "notificationsSent": []
+        }
+      }
+    }
+  },
+  "expectation": {
+    "status": 200,
+    "errors": [],
+    "caseData": {
+      "template": "minimal-appeal-submitted.json",
+      "replacements": {
+        "notificationsSent": [
+          {
+            "id": "1001_FTPA_APPLICATION_DECISION_HOME_OFFICE",
+            "value": "$/[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}/"
+          },
+          {
+            "id": "1001_FTPA_APPLICATION_DECISION_LEGAL_REPRESENTATIVE",
+            "value": "$/[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}/"
+          },
+          {
+            "id": "1001_FTPA_APPLICATION_DECISION_ADMIN_OFFICER",
+            "value": "$/[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}/"
+          }
+        ]
+      }
+    },
+    "notifications": [
+      {
+        "reference": "1001_FTPA_APPLICATION_DECISION_HOME_OFFICE",
+        "recipient": "{$respondentEmailAddresses.respondentEvidenceDirection}",
+        "subject": "Immigration and Asylum appeal: application to appeal to Upper Tribunal granted",
+        "body": [
+          "HMCTS reference: PA/12345/2019",
```

Comment:
You're still checking for some static content. Remove the text "HMCTS reference:". 
Take a look at how the other functional tests are done.

Reply:
All done.